src/test/java/Homework19.java

```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Collection;
+
+public class Homework19 extends BaseTest {
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+
+        provideEmail("prachi.gupta01@testpro.io");
+        providePassword("te$t$tudent1");
+        clickSubmit();
+
+            WebElement playlist = driver.findElement(By.cssSelector(".playlist:nth-child(4)"));
+            playlist.click();
+        Thread.sleep(2000);
+
+            WebElement deletePlaylistBtn = driver.findElement(By.cssSelector(".btn-delete-playlist"));
+            deletePlaylistBtn.click();
+
+            WebElement okBtn = driver.findElement(By.cssSelector(".ok"));
+            okBtn.click();
+        Thread.sleep(2000);
+
+            WebElement notifMsg = driver.findElement(By.cssSelector("div.success.show"));
+            Assert.assertTrue(notifMsg.isDisplayed());
+        Thread.sleep(2000);
+
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)